torch/fx/experimental/fuser.py

```diff
@@ -0,0 +1,45 @@
+from torch.fx import (
+    GraphModule,
+)
+from torch.nn.utils.fusion import fuse_conv_bn_eval
+import torch
+
+# turn foo.bar -> ['foo', 'bar']
```

Comment:
Probably better to add a docstring to this function clarifying that this splits a qualname into its parent path and last atom. As an example `foo.bar.baz` -> ('foo.bar', 'baz')`. Also worth specifying the edge case for a root qualname

Reply:
Shoulda been a reviewer for the initial code :P https://github.com/Chillee/pytorch/blob/fusion/test/fx/quantization.py#L145